cypress/integration/createNewUser.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+import { faker } from "@faker-js/faker";
+
+describe("Testing for add new customer ", () => {
+  before(function () {
+    cy.visitLoginPage();
+    cy.get("a.nav-link").last().click();
```

Comment:
Please put this line of code in login command as this button should be clicked every time for login